src/HDC1080.c

```diff
@@ -48,7 +50,14 @@ float GetTemperature(void)
 	uint8_t regaddr = HDC1080_TEMP_REG;
 	uint8_t regval[2];
 
-	I2C_IF_ReadFrom(HDC1080_DEV_ADDR, &regaddr, 1, regval, 2);
+	I2C_IF_Write(HDC1080_DEV_ADDR, &regaddr, 1, 0);
+
```

Comment:
I made a mistake with this pull request. It doesn't include the prior commits, so you're not seeing all of the source. There is a conversion going on, please take a look [here](https://github.com/jon-whit/airu-firmware/blob/feature/HDC1080-driver/src/HDC1080.c#L47-L67).